specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-10-01-preview/Clusters.json

```diff
@@ -0,0 +1,567 @@
+      },
+      "patch": {
+        "tags": [
+          "Clusters"
+        ],
+        "x-ms-examples": {
+          "ClustersPatch": {
+            "$ref": "./examples/ClustersUpdate.json"
+          }
+        },
+        "operationId": "Clusters_Update",
+        "description": "Updates a Log Analytics cluster.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "clusterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of the Log Analytics Cluster."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ClusterPatch"
+            },
+            "description": "The parameters required to patch a Log Analytics cluster."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The existing cluster was successfully updated. Check provisioningStatus to see detailed status.",
+            "schema": {
+              "$ref": "#/definitions/Cluster"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ClusterErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ClusterProperties": {
+      "properties": {
+        "nextLink": {
+          "description": "The link used to get the next page of recommendations.",
+          "type": "string"
+        },
+        "clusterId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The ID associated with the cluster."
+        },
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The provisioning state of the cluster.",
+          "enum": [
+            "Creating",
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "Deleting",
+            "ProvisioningAccount",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "ClusterEntityStatus",
+            "modelAsString": true
+          }
+        },
+        "isDoubleEncryptionEnabled": {
+          "type": "boolean",
+          "readOnly": false,
+          "description": "Configures whether cluster will use double encryption. This Property can not be modified after cluster creation. Default value is 'true'"
```

Comment:
If this can only be modified on create use "x-ms-mutability" to make that clear.  github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-mutability

Reply:
Done.